include/highfive/bits/H5ConverterEigen_misc.hpp

```diff
@@ -40,31 +40,17 @@ inline size_t compute_total_size(const boost::multi_array<T, Dims>& vec) {
 
 // apply conversion to eigen matrix
 template <typename T, int M, int N>
-struct data_converter<Eigen::Matrix<T, M, N>, void> {
+struct data_converter<Eigen::Matrix<T, M, N>, void>
+    : public container_converter<Eigen::Matrix<T, M, N>> {
 
-    typedef Eigen::Matrix<T, M, N> MatrixTMN;
+    using MatrixTMN = Eigen::Matrix<T, M, N>;
```

Comment:
I would opt for a generic name maybe:
```cpp
using value_type = Eigen::Matrix<T, M, N>
```
(or so)

Reply:
I fix things little by little, at the end those data_converter would be deleted. The generic part is inside `H5Utils` with `inspector`.